### ISSUE TITLE
Stacktrace is empty when using bazel-built glog (aarch64, x86_64)

### ISSUE THREAD

**Issue by dmellosanjay** [google/glog#896]
I have tested `glog` 0.4.0, 0.5.0 and 0.6.0 on `aarch64` and `x86_64` Linux (Ubuntu 18.04) machines and was able to get a stack trace only when I used `glog` 0.5.0 on a x86_64 Linux machine.

Here's an instance of a test on an `aarch64` linux machine using 0.6.0

Test Code:
```
#include <glog/logging.h>
 
void bar(int x) {
  int* invalid_location = 0;
  int temp;

  //temp = *invalid_location;
  CHECK_EQ(x, 1);
}
 
void foo(int x) {
  bar(x + 1);
}

int main(int argc, char** argv) {
  google::InstallFailureSignalHandler();
  foo(1);
}

```
Expected result:
```
WARNING: Logging before InitGoogleLogging() is written to STDERR
F20230223 22:31:24.688732  4806 stack_trace.cpp:8] Check failed: x == 1 (2 vs. 1) 
*** Check failure stack trace: ***
*** Aborted at 1677191484 (unix time) try "date -d @1677191484" if you are using GNU date ***
PC: @                0x0 (unknown)
*** SIGABRT (@0x12c6) received by PID 4806 (TID 0x7f5719ac6740) from PID 4806; stack trace: ***
    @     0x7f57196bd980 (unknown)
    @     0x7f57189b9e87 gsignal
    @     0x7f57189bb7f1 abort
    @           0x21e629 google::logging_fail()
    @           0x21e61d google::LogMessage::Fail()
    @           0x21da78 google::LogMessage::SendToLog()
    @           0x21e2fd google::LogMessage::Flush()
    @           0x221ea9 google::LogMessageFatal::~LogMessageFatal()
    @           0x21c0dd bar()
    @           0x21c116 foo()
    @           0x21c13e main
    @     0x7f571899cc87 __libc_start_main
    @           0x21bf8a _start
    @                0x0 (unknown)
Aborted (core dumped)
```

Actual Result:

```
WARNING: Logging before InitGoogleLogging() is written to STDERR
F20230223 22:31:07.857421  1853 stack_trace.cpp:8] Check failed: x == 1 (2 vs. 1) 
*** Check failure stack trace: ***
*** Aborted at 1677191467 (unix time) try "date -d @1677191467" if you are using GNU date ***
PC: @                0x0 (unknown)
Aborted (core dumped)
```

**Comment by sergiud** [comment]
Can you bisect to determine the commit that introduced the regression?

**Comment by dmellosanjay** [comment]
@sergiud I see this issue was fixed for `x86_64` in https://github.com/google/glog/pull/851. This was broken in release 0.6.0.

Still doesn't work on `aarch64` though.

**Comment by sergiud** [comment]
Did you pinpoint the commit that broke aarch64 symbolization?

**Comment by sergiud** [comment]
Should be fixed by #905.
